lib/postprocessing/image.js

```diff
@@ -346,18 +436,24 @@ async function write(img, outfile) {
     await img.write(outfile);
 }
 
-// exported function
-async function imagePostProcess(rendition, outfile, directories) {
-    const img = await read(rendition.path);
+/**
+ * Runs image post processing on the (intermediate) rendition, to create the final rendition.
+ * @param {*} intermediateRendition existing intermediate rendition created by the worker callback
```

Comment:
nitpick: you could put the class `Rendition` in the type curly braces: `{Rendition}`